rslib/src/backend/mod.rs

```diff
@@ -137,10 +138,40 @@ impl Backend {
             .clone()
     }
 
-    fn web_client(&self) -> &Client {
-        // currently limited to http1, as nginx doesn't support http2 proxies
-        self.web_client
-            .get_or_init(|| Client::builder().http1_only().build().unwrap())
+    fn set_custom_cert(&self, cert_str: String) -> Result<()> {
+        let cert_conv = Certificate::from_pem(cert_str.as_bytes());
+
+        if cert_conv.is_ok() {
```

Comment:
Instead of a check + unwrap, you should use if let. There are a few similar cases that also need updating.